package.json

```diff
@@ -29,6 +29,7 @@
         "bundle-collapser": "^1.3.0",
         "caporal": "1.0.0",
         "chalk": "^2.4.2",
+        "D": "^1.0.0",
```

Comment:
Why?
`This package has been deprecated`

Reply:
A guess: `npm install D` 😅

Reply:
I've updated the Dockerfile to use `node:20-alpine` to minimize the CVEs to 2 medium ones. It also shrinks the image by 60 MB.

The `D` has been removed.

I also noticed a warning that `babel-eslint` was no longer updated, so I switched to `@babel/eslint-parser` to make any future updates easier.